compiler/src/dotty/tools/MainGenericRunner.scala

```diff
@@ -204,6 +191,8 @@ object MainGenericRunner {
         }
         errorFn("", res)
       case ExecuteMode.Script =>
+        val targetScriptPath: String = settings.targetScript.toString.replace('\\', '/')
+        System.setProperty("script.path", targetScriptPath)
```

Comment:
Changes are already introduced in scripting [Main](https://github.com/lampepfl/dotty/blob/master/compiler/src/dotty/tools/scripting/Main.scala). Please decide which place is correct for setting up this property and remove one.

Reply:
Wow, I'm a bit surprised, since I only added it to [MainGenericRunner](https://github.com/lampepfl/dotty/blob/master/compiler/src/dotty/tools/MainGenericRunner.scala) because a script was unable to see the property.  I'll clean it up.

Reply:
No problem